generators/structure/index.js

```diff
@@ -0,0 +1,36 @@
+const yeoman = require('yeoman-generator');
+const yosay = require('yosay');
+const _includes = require('lodash/includes');
+
+module.exports = yeoman.Base.extend({
+  prompting: {
+    structure() {
+      const self = this;
+
+      this.log(yosay(
+        'Create Folder Structure'
+      ));
+      return this.prompt([{
```

Comment:
@riteshe63 The only prompt here will be asking if we need to add basic folder structure? It will be a yes / no question not multiple choice. See https://github.com/SBoudrias/Inquirer.js#questions